src/components/TopStory.vue

```diff
@@ -32,9 +32,14 @@
 <template>
 
   <article class="card_top-story" v-if="articleArray.article_title">
-    <a class="scoop-source-url article-image-main" :href="articleArray.article_url" target="_blank">
-      <span class="article-image-wraper" :style="{ 'background-image': 'url(' + articleArray.article_image + ')' }"></span>
+    <a
+      :href="articleArray.article_url"
+      :style="{ 'background-image': 'url(' + articleArray.article_image + ')' }"
+      target="_blank"
+      class="module-tile image-only"
```

Comment:
ditto here with module. I jus dont think it makes sense. In my mind a module is a very different thing.

Reply:
Please explain your mind, then! I'm open to anything to help clarify "changes to the card. "what card? oh, not that card, the card inside that card, that's also inside the card."